request.js

```diff
@@ -1411,6 +1405,19 @@ Request.prototype.destroy = function () {
     self.response.destroy()
   }
 }
+Request.prototype.enableUnixSocket = function () {
```

Comment:
Not the best place to put this function. Notice the comment above - *Streams API*